source/text/fontloader.ts

```diff
@@ -251,56 +253,56 @@ export class FontLoader {
     }
 
     /**
-     * Asynchronously loads a fnt-file and a png-file of the same name, to create a font face from them.
+     * Asynchronously loads a fnt-file and referenced pages to create a font face from them.
      * @param context - Valid context to create the object for.
-     * @param filename - The path to the fnt-file.
+     * @param uri - URI linking the fnt-file that should be loaded. Data URI not yet supported.
      * @param headless - Boolean for headless mode.
      * @param onImageLoad - Callback is called when the glyph atlas is loaded.
      */
-    static async load(context: Context, filename: string, headless: boolean): Promise<FontFace> {
+    static async load(context: Context, uri: string, headless: boolean): Promise<FontFace> {
         const fontFace = new FontFace(context);
 
         this._valid = true;
 
         let text;
         try {
-            text = await fetchAsync(filename, '', (text) => text);
+            text = await fetchAsync(uri, '', (text) => text);
         } catch (e) {
             /* promise rejected */
-            log(LogLevel.Warning, `Could not load font file. filename is: ${filename}`);
+            log(LogLevel.Warning, `loading font file '${uri}' failed`);
             this._valid = false;
         }
 
         /* promise fulfilled */
         const lines = text.split('\n');
 
         const promises = [];
-        for (const l of lines) {
+        for (const line of lines) {
             if (!this._valid) {
                 break;
             }
-            let line = l.split(' ');
-            const identifier = line[0];
-            line = line.slice(1);
+            let values = line.split(' ');
```

Comment:
values could look like that:
`[ "first=8222", "second=199", "amount=-7.17\r" ]`

Some names I suggest that could be more suitable for `values`:
`valueFields`, `attributes`, or `keyValues`  (the last one would be consistent with the function `readKeyValuePairs`).

Reply:
`values` is also already used in `processInfo()`. There, they would be the "values of values", so I'd like to encourage to rename the `values` here to something different.